Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
-# unreleased
+# 2020-02 -- v4.3.0
+- [changed] Deprecated Messaging Direct Channel and upstream API. Please use APNS channel instead for data message delivery. Use FireStore, real time database or other services for real time updates. The deprecated API will be removed in Firebase 7. (#4710)
```

Comment:
We never really "productized" this feature with capital letters.  Suggest just "Deprecated FCM direct messaging via `shouldEstablishDirectChannel`. Instead, use APNs for downstream message delivery. "

Reply:
Also suggest separating direct channel and upstream.  Would that work?  Then you'd have also:

- [changed] Deprecated upstream messaging API.  For realtime updates, use Cloud Firestore, Realtime Database, or other services.  The deprecated . . . "